cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/.github/workflows/constraints.txt

```diff
@@ -0,0 +1,5 @@
+pip==20.3.3
```

Comment:
we should add this to cookietemple dependabot config as well

Reply:
Those constraints are used for the GH workflows, not dependabot.

Reply:
Ând they dont need to be kept up to date?

Reply:
Dependabot actually picked them up (somehow).